scripts/format.sh

```diff
@@ -19,8 +19,12 @@ install_py_packages $py
 python_sources=$(LANG=C find scripts src/bindings/python -type f ! -name "*.md" ! -name "PKG-INFO" -exec file {} \; | grep -F "Python script" | sed 's/:.*//')
 
 # Python formatter
-echo ${python_sources}
-${py} -m black --config pyproject.toml ${python_sources}
+echo "${python_sources}"
+${py} -m black \
+  --line-length 105 \
+  --skip-string-normalization \
+  --extend-exclude "/(\..*|dist|wheelhouse|.*build|__pycache__|.*\.toml|.*\.egg-info)/" \
```

Comment:
ported from the pyproject.toml, which might now not exist, unless one calls e.g.

```
cmake -DPYVALHALLA_NAME=pyvalhalla -P cmake/ValhallaConfigPyProject.cmake
```